paasta_tools/long_running_service_tools.py

```diff
@@ -57,6 +57,17 @@ class LongRunningServiceConfigDict(InstanceConfigDict, total=False):
 
 class ServiceNamespaceConfig(dict):
 
+    def get_healthcheck_mode(self) -> str:
```

Comment:
I'm really surprised this is in the ServiceNamespaceConfig. I thought `healthcheck_mode` was only something we let people set in their marathon config?

Reply:
it's in https://paasta.readthedocs.io/en/latest/yelpsoa_configs.html#smartstack-yaml ?

Reply:
Yeah, it's possible to have a separate healthcheck mode and mode for smartstack -- we can do just TCP checking for an HTTP smartstack service, or do HTTP checking for a TCP smartstack service.